### PR TITLE
test: run a11y suite in circleci

### PR DESCRIPTION
**Pull request by emilyjablonski**
In porting over, we missed running the accessibility suite in CircleCi.

To test this PR, ensure the tests run and pass. In getting the suite to work, I realized some suites were silently failing due to a config issue, and once they were running some tests were failing, so it involves a few a11y fixes as well. One of the errors I had to explicitly ignore due to it being a third party issue with ag-table.

### CODE REVIEW

File: src/tables/AgTable.stories.tsx [hunk 1]
```diff
@@ -73,3 +73,17 @@ export const Default = () => {
     />
   )
 }
+
+// The external library we are using is throwing an a11y error:
```
Comment (ludtkemorgan) [inline src/tables/AgTable.stories.tsx:77]: Can you name the library here? that way we can potentially track the issue

**Review by ludtkemorgan — APPROVED** [review]
Thanks for addressing this!

File: src/tables/AgTable.stories.tsx (hunk 1 above)
Comment (emilyjablonski) [reply src/tables/AgTable.stories.tsx]: Updated!

**Review by ColinBuyck — APPROVED** [review]
LGTM 🏂

**Pull request merged**